app/javascript/.eslintrc.js

```diff
@@ -22,9 +22,12 @@ module.exports = {
       },
     ],
     'import/prefer-default-export': 'off',
+    'no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
```

Comment:
We still want the linter to complain about unused args or vars, but in the case of unused args that we might want to keep, prefix them with an `_`.

Reply:
Another option is to just remove them. I'll let the team decide on that one.

![](https://media.giphy.com/media/wIji5V2SF2jNm/giphy.gif)

Reply:
We can keep it with the prefix.